.github/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/pi-hole/ftl-build:v2.10 AS builder
+FROM ghcr.io/pi-hole/ftl-build:v2.11 AS builder
```

Comment:
This change was unintentional here but if you don't mind, I'd like to avoid the overhead of removing this again and opening a separate PR.